### PR TITLE
[IMP] l10n_no: translation

### PR DESCRIPTION
**Pull request by malb-odoo**
Before this PR, all this localisation was written in Norwegian, but all the localisation have to be written in english and then translated back in the native language. This PR correct that.

Task-id: 3149392

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by h4818 — COMMENTED** [review]
LGTM

File: addons/l10n_no/data/account.account.template.csv [hunk 1]
```diff
@@ -1,473 +1,473 @@
...
+"chart6300","6300","Rent premises","expense","","False","no_chart_template"
+"chart6320","6320","Renovation, water, sewage etc.","expense","","False","no_chart_template"
+"chart6340","6340","Light, heat","expense","","False","no_chart_template"
+"chart6360","6360","Cleaning","expense","","False","no_chart_template"
+"chart6390","6390","Other cost premises","expense","","False","no_chart_template"
+"chart6400","6400","Rental of operating assets","expense","","False","no_chart_template"
+"chart6410","6410","Rental of fixtures","expense","","False","no_chart_template"
+"chart6420","6420","Rent computer systems","expense","","False","no_chart_template"
+"chart6430","6430","Rent other office machines","expense","","False","no_chart_template"
+"chart6440","6440","Rent means of transport","expense","","False","no_chart_template"
```
Comment (vin-odoo) [inline addons/l10n_no/data/account.account.template.csv:355]: Rent/ rental
Align these names to use the same terms

**Review by vin-odoo — CHANGES_REQUESTED** [review]
annen = other
andre = others

**Review by vin-odoo — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**